src/web/app/pages-monitoring/logs-page/logs-page.component.html

```diff
@@ -98,7 +98,9 @@ <h1>Logs</h1>
             <label for="source-file" class="fw-bold">Source location file</label>
             <input id="source-file" class="form-control col-12" [(ngModel)]="formModel.filters.sourceLocation!.file">
           </div>
-          <div class="col-12 mt-2">
+        </div>
+        <div class="col-sm-6 col-1">
+          <div class="col-12">
```

Comment:
I believe these lines (and the similar lines below) are causing the input fields to have a different layout, particularly for desktop view. I personally think we should keep the layout the same.